examples/vector_collection/vector_collection.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from hazelcast import vector
+from hazelcast.core import HazelcastJsonValue
+from hazelcast.vector import Type, Vector
+
+logging.basicConfig(level=logging.DEBUG)
+
+from hazelcast.client import HazelcastClient, Config
+from hazelcast.proxy.vector_collection import Document
+
+
+def main():
+    # Connect to the cluster
+    cfg = Config()
+    cfg.cluster_members = ["localhost:5701"]
+    client = HazelcastClient(cfg)
+
+    vc_name = "my-vector-collection"
+
+    # Create the VectorCollection
+    indexes = [
+        vector.IndexConfig(name="default-vector", metric=vector.Metric.COSINE, dimension=2),
```

Comment:
nit: 

the following can be better in terms of readability & consistency

```suggestion
       IndexConfig(name="default-vector", metric=Metric.COSINE, dimension=2),
```

Reply:
https://github.com/hazelcast/hazelcast-python-client/pull/674/commits/3c27230ad7d9777be1698db0d6ceb00121844ba8